browser_tests/tests/vueNodes/interactions/node/imagePreview.spec.ts

```
@@ -1,11 +1,11 @@
 import { expect } from '@playwright/test'
 
-import type { ComfyPage } from '../../../../fixtures/ComfyPage'
-import { comfyPageFixture as test } from '../../../../fixtures/ComfyPage'
+import type { ComfyPage } from '@e2e/fixtures/ComfyPage'
+import { comfyPageFixture as test } from '@e2e/fixtures/ComfyPage'
```

ESLint

[error] Unable to resolve path to module '@e2e/fixtures/ComfyPage'.

(import-x/no-unresolved)